QA.md

```diff
@@ -62,6 +66,16 @@
         1. Send 1 HTR to your own address (it will ask for the biometry again)
         1. Disable biometry
         1. Lock wallet and it must require you to type your PIN Code.
+    1. Test safe Biometry mode
+        1. Go to unleash and enable the `safe-biometry-mode.rollout` for this wallet.
+        1. Enable biometry (either Fingerprint, FaceID or TouchID depending on the device, it will require your pin)
```

Comment:
Biometry will already be enabled at this point, maybe make the last step from the previous section disable the biometry

Reply:
The second last step of the last section is to disable the biometry, so the biometry is not enabled here.
And the first step of the last section is to disable the feature flag, so we need to start with the feature flag enable.

Reply:
You're right